package.json

```diff
@@ -182,6 +184,7 @@
         "eslint-plugin-vue": "~8.7.1",
         "favico.js": "~0.3.10",
         "get-port-please": "^3.1.1",
+        "globals": "^15.13.0",
```

Comment:
https://github.com/louislam/uptime-kuma/pull/5444/files

This doesn't present in the old pull request. Is it required?